chapters/cli/exit.rst

```diff
@@ -1,8 +1,25 @@
+====================
 Gracefully Exit Jina
 ====================
+---------------------------------------------------------------------------------
+Summary: In this section, you will learn best practices to shut down a Flow and exit Jina correctly.
+---------------------------------------------------------------------------------
+
+Feature description and expected outcome
+----------------------------------------
+Jina provides several methods to exit gracefully.
+
+Through these ways the Flow will terminate and release all resources.
+
+Before you start:
+-----------------
+Make sure you install Jina via `Installation <https://docs.jina.ai/chapters/install/os/index.html>`_.
+
+Implementation:
+---------------
 
 In Python
----------
+^^^^^^^^^
 
 If you use ``with`` scope to start a Flow, then all resources (including Pods of all kinds) of the Flow will be released after your move out from the scope.
```

Comment:
Typo in "your" it should be "you" right?

Reply:
✔️